packages/scrollable-image/makefile

```diff
@@ -0,0 +1,54 @@
+# WARNING:
```

Comment:
The makefile under single package also need update:
https://github.com/twreporter/twreporter-npm-packages/blob/master/packages/react-article-components/makefile

It will take all the commands of targets named as `xxxx-default` in `dev/base.makefile`, and put them as targets named as `xxxx` in this file.

You can override the commands in targets `dev`, `build`, ... if needed.